src/style.css

```diff
@@ -19,16 +37,76 @@ li {
   gap: 2rem;
 }
 
+button {
+  cursor: pointer;
+  border: none;
+  padding: 10px;
+  background-color: orange;
+  border-radius: 5px;
+}
+
```

Comment:
- [ ] Good job with the design. Currently, the trophy image is broken, kindly fix it. Let's try to maintain the wireframe layout. I think it would be better if you align the title `Leaderboard` to the left side and please align the submit button to the right side. ✍️

![image](https://user-images.githubusercontent.com/17243325/216426048-bfa3f4ae-000d-4b61-902d-ea6ff6b0b19e.png)